committee-steering/governance/sig-governance.md

```diff
@@ -8,7 +8,10 @@ In order to standardize Special Interest Group efforts, create maximum transpare
 - Meet regularly, at least for 30 minutes every 3 weeks, except November and December
 - Keep up-to-date meeting notes, linked from the SIG's page in the community repo
 - Record meetings and make them publicly available on a YouTube playlist
-- Report activity in the weekly community meeting at least once every quarter
+- Report activity with the community at least once a quarter via any of the following means:
```

Comment:
If we are providing multiple avenues for updates, should contribex help collect those together and send out a mailing each quarter with them? Would this give us any benefit? Will contribex continue to track these updates, and raise to steering any SIGs that are not in compliance?

Reply:
I pushed a PR to make it more explicit that k-dev should be the final destination for a status update.